tor/core/inbox.py

```diff
@@ -23,6 +22,16 @@
 ]
 
 
+def send_reply_to_slack(item, config):
```

Comment:
Do we want to instead rename this method to "forward_to_slack"? It seems more appropriate in the given context.